data/hyperOutportBlocks/hyperOutportBlock.proto

```diff
@@ -18,11 +20,23 @@ message BlockHashRequest {
   string hash = 1;
 }
 
+message BlockHashStreamRequest {
```

Comment:
add a TODO for moving rpc related structs to a separate file, since we import this file in `firehose-multiversx` repo (currently), and we don't need them there, we need only outport block related structus